corehq/apps/geospatial/const.py

```diff
@@ -136,3 +136,4 @@
 
 # Case property to identify cases assigned through disbursement on the Case Management Page
 ASSIGNED_VIA_DISBURSEMENT_CASE_PROPERTY = 'commcare_assigned_via_disbursement'
+INDEX_ES_TASK_HELPER_BASE_KEY = 'geo_cases_index_cases'
```

Comment:
nit: the name felt other way around, I'd expect it to be `ES_INDEX_TASK_HELPER_BASE_KEY`.

Reply:
That makes sense, I'll rename this as you suggested.

Reply:
Addressed in [15c6383](https://github.com/dimagi/commcare-hq/pull/35126/commits/15c6383da814598148bcc6b9aed6827cc49e4493).